.gitignore

```diff
@@ -190,5 +190,6 @@ test/sleep.json
 .vscode
 
 /tmp
-.superduperdb
-output/
+/.superduperdb
+/output/
```

Comment:
?

Reply:
`/` at the beginning forces the root directory.

Reply:
https://stackoverflow.com/questions/3637660/how-to-exclude-file-only-from-root-folder-in-git